libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -522,11 +523,11 @@ void AsmAnalyzer::warnOnInstructions(solidity::Instruction _instr, SourceLocatio
 			"the Metropolis hard fork. Before that it acts as an invalid instruction."
 		);
 
-	if (_instr == solidity::Instruction::JUMP || _instr == solidity::Instruction::JUMPI)
+	if (_instr == solidity::Instruction::JUMP || _instr == solidity::Instruction::JUMPI || _instr == solidity::Instruction::JUMPDEST)
 		m_errorReporter.warning(
 			_location,
 			"Jump instructions are low-level EVM features that can lead to "
```

Comment:
We could also write `Jump instructions and labels are low-level...` to combine all issues.